picard/coverart/image.py

```diff
@@ -27,16 +27,17 @@
 import tempfile
 from hashlib import md5
 
-from PyQt5.QtCore import QUrl, QObject, QMutex
+from PyQt5.QtCore import (QMutex,
```

Comment:
Just a thing, i prefer following syntax:
```python
from PyQt5.QtCore import (
    QMutex,
    QObject,
    QUrl,
)
```

Reason: if you change the list, ie. sort it, or insert/delete lines, diffs are much easier to read. Sorting is easier too (select all, sort). The extra comma is legit in Python, and convenient when you re-order.
Also it requires only 4 spaces to indent.